docs/get-started.md

```diff
@@ -51,7 +51,7 @@ Also update the Babel configuration in the `package.json` file:
 
 4. Install `react-rails`:
 ```bash
-$ bundle add 'react-rails' --strict
+$ bundle add 'react-rails' --strict --version '>=3.1.1'
```

Comment:
We should use "exact" version.

Reply:
But I see one point. `--strict` doesn't count the patch version and when we have fixes and bump the patch value in the version, `--strict` doesn't help.

If there is any better proposal (than manually fix the version) please share.

Reply:
I'm not seeing the value of any changes. What am I missing?